README.md

```diff
@@ -12,7 +12,38 @@ This module is used by the CI tests of the different Five Bells components. This
 npm install --save-dev five-bells-integration-test
 ```
 
-## Usage
+## Usage (with Docker)
+Get the Docker image for the Five Bells integration tests,
+and save yourself a lot of configuration and slow building steps on
+your laptop:
+```sh
+docker pull michielbdejong/five-bells-integration-test
```

Comment:
I'll update this, here and in other places, to interledgerjs/five-bells-integration-test once that's published